docs/source/tutorials/advanced/datumaro.rst

```diff
@@ -0,0 +1,7 @@
+###########################################
+Convert a custom user dataset with Datumaro
+###########################################
+
+<Guidelines for writing>
+
+This tutorial illustrates how to build a user dataset for each task. The tutorial also demonstrates how to utilize datumaro in conjunction with OTX using a real-world example.
```

Comment:
Please add a reference to Datumaro. Also, I suggest using a capital letter for the product naming